python/pylibcudf/pylibcudf/tests/test_expressions.py

```diff
@@ -47,3 +46,35 @@ def test_columnnameref_construction():
 )
 def test_astoperation_construction(kwargs):
     plc.expressions.Operation(**kwargs)
+
+
+def test_evaluation():
+    table_h = pa.table({"a": [1, 2, 3], "b": [4, 5, 6], "c": [7, 8, 9]})
+    lit = pa.scalar(42, type=pa.int64())
+    table = plc.interop.from_arrow(table_h)
+    # expr = abs(b * c - (a + 42))
```

Comment:
We don't need to do it in this PR, but do you think we should migrate the logic from https://github.com/rapidsai/cudf/blob/branch-24.12/python/cudf/cudf/core/_internals/expressions.py into pylibcudf to make it easier to construct expressions, or do you think the string->expr mapping there has too much ambiguity and we should continue to require explicit construction?

Reply:
I think because the string is untyped, and the expressions need to be typed, I would not do this.